modules/ROOT/pages/tutorial/access-control.adoc

```diff
@@ -934,24 +1961,31 @@ If you check what privileges Daniel has now, it is the combination of the two ro
 
 [source, cypher, role=systemdb]
 ----
-neo4j@system> SHOW USER daniel PRIVILEGES AS COMMANDS;
+SHOW USER daniel PRIVILEGES AS COMMANDS;
 ----
+
+.Result
+[queryresult]
 ----
 +---------------------------------------------------------------------+
 | command                                                             |
 +---------------------------------------------------------------------+
-| "GRANT ACCESS ON HOME DATABASE TO $role"                            |
+| "DENY CREATE ON GRAPH `healthcare` RELATIONSHIP DIAGNOSIS TO $role" |
 | "GRANT ACCESS ON DATABASE `healthcare` TO $role"                    |
-| "GRANT EXECUTE PROCEDURE * ON DBMS TO $role"                        |
+| "GRANT ACCESS ON DATABASE `neo4j` TO $role"                         |
```

Comment:
Maybe we should drop the database `neo4j`, since it is not relevant to this tutorial?

Reply:
I guess we just don't need to mention it at all.

Reply:
Maybe because the default home database is `neo4j`, all these privileges are automatically granted to users.

Reply:
yes, if we drop the `neo4j` database you can make the output a little bit clearer without having the `neo4j` db.

Reply:
Hey @HannesSandberg, instead of dropping `neo4j`, I decided to change the default database to `healthcare`. This is just one step after creating the database. Otherwise, I had to add a step for each user to log into `healthcare`; otherwise, Neo4j throws an authentication error.

Reply:
Nice solution 👍